linopy/io.py

```diff
@@ -729,7 +728,7 @@ def to_gurobipy(m: Model, env: Union[Any, None] = None) -> Any:
     return model
 
 
-def to_highspy(m: Model) -> Highs:
```

Comment:
Could you use a the `TYPE_CHECKING` check instead?
As done in here
https://github.com/PyPSA/linopy/blob/f25f208daaff1ef5f5aae916f937eca4df0685fa/linopy/constraints.py#L61-L62

Reply:
I was thing about it. It should work provided one uses dev set of optional dependencies, which I assume developers are using.

Reply:
It should work without optional dependencies as well, when wrapped with the type checking flag

Reply:
Done